include/cuda/std/array

```diff
@@ -9,6 +9,10 @@
 #ifndef _CUDA_ARRAY
 #define _CUDA_ARRAY
 
+#ifndef __CUDACC_RTC__
+# include <cstdlib>
```

Comment:
Out of curiosity, what happens when using NVRTC with things like this? Will the functionality that depends on stuff from `<cstdlib>` just not work?

Reply:
Usually NVRTC defines its own equivalent. It's all just `__device__` only.